src/main/java/com/artipie/rpm/Repomd.java

```diff
@@ -121,103 +128,106 @@ private CompletableSource performUpdate(
         final Act act,
         final Path temp
     ) {
-        return Single.fromCallable(() -> Files.createTempFile("x", ".data"))
-            .flatMapCompletable(
-                file -> {
-                    final XML xml = new XMLDocument(temp.toFile())
-                        .registerNs("ns", "http://linux.duke.edu/metadata/repo");
-                    final List<XML> nodes = xml.nodes(
-                        String.format("/ns:repomd/data[type='%s']", type)
+        return Single.fromCallable(() -> Files.createTempFile("x", ".data")).flatMapCompletable(
```

Comment:
@g4s8 What about temp file deletion afterwards??? I am 100% sure, but looks like the temp file needs to be deleted in the end.

Reply:
@fanifieiev I'm not sure about this it, these changes were introduced before my commit, so I didn't change it, since it's out of scope for current task.

Reply:
@g4s8 OK